bom/runtime/pom.xml

```diff
@@ -734,6 +734,11 @@
                 <artifactId>quarkus-spring-security</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-spring-scheduling</artifactId>
+                <version>${project.version}</version>
+            </dependency>
```

Comment:
The deployment artifact should also be added to the deployment bom.

Reply:
yes, ok. Thx!